.gitignore

```diff
@@ -0,0 +1,131 @@
+*.DS_Store
```

Comment:
Maybe we do not need the `.gitignore`

Reply:
I like having an ignore file as an easy way of ensuring ancillary/extraneous files that get created in the repo directory don't get committed into the project. e.g. Calling `git commit -am "some message"` when there's random non-source files in the repo.

Are there any specific reason(s) why your find the ignore file unnecessary?